v2.0/resourcepacks/create_astral_translation/assets/projectiledamage/lang/zh_cn.json

```
@@ -0,0 +1,5 @@
+{
+    "attribute.name.generic.projectile_damage" : "弹射物伤害",
+    "item.modifiers.both_hands": "当手持时：",
+    "effect.projectiledamage.impact": "冲击"
```

```suggestion
    "effect.projectiledamage.impact": "强击"
```
考虑到“冲击”和原版的“冲击”附魔（Punch）重复，改为“强击”